### ISSUE TITLE
 Index creation failed: mapper_parsing_exception: Failed to parse mapping : Field [_id] is defined twice 

### ISSUE THREAD

**Issue by skonsoft** [sineflow/ElasticsearchBundle#3]
Hello,

I followed exactly the docs to create a very basic example document, but it won't parse it with errir:

> Index creation failed: mapper_parsing_exception: Failed to parse mapping [country]: Field [_id] is defined twice in [country]

My configuration is:

```
sineflow_elasticsearch:
    connections:
        default:
            hosts: [127.0.0.1:9200]           
    indices:
        countries:
            name: afariat_country
            connection: default
            types:
                - SkonsoftModelBundle:Country
```

And my document definition:

```
<?php

namespace Skonsoft\ModelBundle\Document;

use Sineflow\ElasticsearchBundle\Document\AbstractDocument;
use Sineflow\ElasticsearchBundle\Annotation as ES;

/**
 * @ES\Document(type="country")
 *
 * @author skander
 */
class Country extends AbstractDocument
{
    /**
     * @var string
     * 
     * @ES\Property(name="name", type="string")
     *
     */
    public $name;

    /**
     * @var string
     *
     * @ES\Property(name="isoCode", type="string")
     */
    public $isoCode;

}

```

When i run command:
`app/console sineflow:es:index:create countries`

I got this error:

> Index creation failed: mapper_parsing_exception: Failed to parse mapping [country]: Field [_id] is defined twice in [country]

Have you any idea it won't work ?

Thank you.

**Comment by stevepop** [comment]
Hi, do you already have this index created? If so, delete that index first then try re-running the command. This looks like it is attempting to create a mapping that already exists.

**Comment by skonsoft** [comment]
Hi, 
No indexes was created.

Here is the output of the query match_all:

```
{
   "took": 1,
   "timed_out": false,
   "_shards": {
      "total": 0,
      "successful": 0,
      "failed": 0
   },
   "hits": {
      "total": 0,
      "max_score": 0,
      "hits": []
   }
}
```

Did the parser reads my entities (doctrine entities) configuration ? if then, i  have set id as protected and not public as suggested the docs. I have putted the document in separate namespace as docs suggested and i did not added annotations to my entities.

**Comment by skonsoft** [comment]
Here is the logs of Elasticsearch when i launch the command:

```
[2016-07-08 15:09:06,723][DEBUG][action.admin.indices.create] [node-1] [afariat_country_20160708150906] failed to create
MapperParsingException[Failed to parse mapping [country]: Field [_parent] is defined twice in [country]]; nested: IllegalArgumentException[Field [_parent] is defined twice in [country]];
    at org.elasticsearch.cluster.metadata.MetaDataCreateIndexService$1.execute(MetaDataCreateIndexService.java:332)
    at org.elasticsearch.cluster.ClusterStateUpdateTask.execute(ClusterStateUpdateTask.java:45)
    at org.elasticsearch.cluster.service.InternalClusterService.runTasksForExecutor(InternalClusterService.java:468)
    at org.elasticsearch.cluster.service.InternalClusterService$UpdateTask.run(InternalClusterService.java:772)
    at org.elasticsearch.common.util.concurrent.PrioritizedEsThreadPoolExecutor$TieBreakingPrioritizedRunnable.runAndClean(PrioritizedEsThreadPoolExecutor.java:231)
    at org.elasticsearch.common.util.concurrent.PrioritizedEsThreadPoolExecutor$TieBreakingPrioritizedRunnable.run(PrioritizedEsThreadPoolExecutor.java:194)
    at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1145)
    at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:615)
    at java.lang.Thread.run(Thread.java:745)
Caused by: java.lang.IllegalArgumentException: Field [_parent] is defined twice in [country]
    at org.elasticsearch.index.mapper.MapperService.checkFieldUniqueness(MapperService.java:455)
    at org.elasticsearch.index.mapper.MapperService.merge(MapperService.java:335)
    at org.elasticsearch.index.mapper.MapperService.merge(MapperService.java:289)
    at org.elasticsearch.cluster.metadata.MetaDataCreateIndexService$1.execute(MetaDataCreateIndexService.java:329)

```

**Comment by pmishev** [comment]
Apparently some breaking change has been introduced in Elasticsearch 2.2.0 that causes this error. Versions before that are fine. I'll investigate further and find a way to fix that in the next few days.

**Comment by pmishev** [comment]
The issue is fixed now in the new release v0.7.0
